src/anemoi/inference/runner.py

```diff
@@ -36,10 +36,10 @@
 
 
 def forcing_and_constants(source, date, param):
-    import climetlab as cml
+    import earthkit
```

Comment:
Can you change this to the following, it is the style we use accross anemoi:

`import earthkit.data as ekd`

Reply:
Shall do, twas wondering if there was a shorthand